src/frameworks/utils.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Whether the given string is a URL.
+ */
+export function isUrl(url: string): boolean {
+  return url.startsWith("http");
```

Comment:
we should make sure this is a protocol, /^https?:\/\//, so say `httpendpoint/foo/bar` wouldn't trip it

Reply:
@jamesdaniels 🤦‍♂️ right, thanks! I also added the example you mentioned as test cases